packages/components/organisms/f-header/src/components/UserNavigationPanel.vue

```diff
@@ -61,6 +61,17 @@ export default {
         returnLogoutUrl: {
             type: String,
             default: ''
+        },
+        countrySelectorIsClosedOnMobileView: {
```

Comment:
If this is a Boolean, I'd try and start it with `is` or `has`. Does this just define if the Country selector is closed, or if it should be closed on mobile (it's a little ambiguous atm)

```suggestion
        isCountrySelectorClosedOnMobileView: {
```

Reply:
`countrySelectorIsClosedOnMobileView` was an existing computed prop so I have renamed as well.